manifest.json

```
@@ -30,6 +30,18 @@
                 "name": "domain",
                 "type": "domain"
             },
+            {
+                "name": "username",
+                "type": "user"
```

Note that the defacto standard is to name this question "admin" instead of "username". That way, the core will automatically create the proper human-readable question "Choose an admin user for this app blahblah" (otherwise you have to specify it yourself in the manifest". Also the CI will work better by automatically filling this argument (otherwise you have to specify it yourself in check_process).

```suggestion
                "name": "admin",
                "type": "user"
```

---

Thanks! changed.